### PR TITLE
add missed lib in requirements.txt and trivial fix in import lib

### PR DESCRIPTION
**Pull request by even-even**
### Describe the Pull Request

```
1) Add missing lib in requirements.txt
2) Fixed trivial ambiguity in the enumeration of modules
```

### Checklist for PR

- [ ] Run MobSF unit tests and lint `tox -e lint,test`.
- [ ] Tested Working on Linux, Mac, Windows, and Docker
- [ ] Make sure build is passing on your PR. [Build Status](https://travis-ci.com/MobSF/Mobile-Security-Framework-MobSF/pull_requests)

### Additional Comments (if any)

```
DESCRIBE HERE
```

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -19,6 +19,8 @@ asn1crypto==1.4.0
 oscrypto==1.2.1
 distro==1.5.0
 IP2Location==8.4.1
+mitmproxy==5.2
```
Comment (ajinabraham) [inline requirements.txt:22]: These are installed by other components like `http-tools`

Comment (even-even) [reply requirements.txt]: Delete mitmproxy lib

**Comment by ajinabraham** [comment]
I don't see any significant addition in this PR other than the formatting of `)`
We prefer to use 
```
from x import (
    a,
    b,
    c,
)
```

**Comment by even-even** [comment]
> I don't see any significant addition in this PR other than the formatting of `)`
> We prefer to use
> 
> ```
> from x import (
>     a,
>     b,
>     c,
> )
> ```

Sorry.
I rolled back my changes related to codestyle.

**Comment by ajinabraham** [comment]
I am closing this since there are no changes.

**Pull request closed without merging**